qt/python/mantidqt/mantidqt/widgets/sliceviewer/views/dataview.py

```diff
@@ -553,7 +553,7 @@ def get_default_scale_norm(self):
             return scale
 
         if self.conf.has(SCALENORM):
-            scale = self.conf.get(SCALENORM)
+            scale = self.conf.get(SCALENORM, type=str)
 
         if scale == 'Power' and self.conf.has(POWERSCALE):
             exponent = self.conf.get(POWERSCALE)
```

Comment:
I think this line needs `type=str` as well

Reply:
Do you mean line 556 and line 559?

Reply:
Yes, there was a [recent change](https://github.com/mantidproject/mantid/pull/34089) to make the `type` mandatory when getting config options. Looks like these were accidentally missed

Reply:
Should be all squared up. I added some release notes as well.

Reply:
Do we know why a test didn't catch this?

Reply:
I think this doesn't need release notes as the problem is not in v6.4

There may be a test missing from `test_sliceviewer_dataview` https://github.com/mantidproject/mantid/blob/main/qt/python/mantidqt/mantidqt/widgets/sliceviewer/test/test_sliceviewer_dataview.py